molecule/azkabanwebsolo/create.yml

```diff
@@ -0,0 +1,70 @@
+---
+- name: Create
```

Comment:
Good job! But the README file specifies that this role is tested using Vagrant. Could you update the documentation? :)

Reply:
Done